DiamondLightSource/python-artemis
# 40: Added code to trigger analysis and associated tests

Fixes #40

To test:
* Confirm unit tests pass
* Confirm that start/end functions perform the same job as `RunAtStartOfCollect-i03.sh` and `RunAtEndOfCollect-i03.sh` (both in `dls_sw\apps\mx=scripts\bin`) Note that they both call https://github.com/DiamondLightSource/python-zocalo/blob/main/src/zocalo/cli/go.py but with different args
